pinecone/client.go

```diff
@@ -218,6 +587,39 @@ type CreateServerlessIndexRequest struct {
 	Region    string
 }
 
+// CreateServerlessIndex creates and initializes a new serverless index via the specified Client.
+//
+// Parameters:
+// 	- ctx: A context.Context object controls the request's lifetime, allowing for the request
+//  to be canceled or to timeout according to the context's deadline.
+//  - in: A pointer to a CreateServerlessIndexRequest object.
+//
+// Returns a pointer to an Index object. In case of failure, it returns nil and the error encountered.
+//
+// Example:
+//  ctx := context.Background()
+//
+//  clientParams := pinecone.NewClientParams{
+//    ApiKey:    getEnvVars("PINECONE_API_KEY"),
```

Comment:
`getEnvVars` throws an error for me. `os.Getenv` works

Reply:
Yeah `getEnvVars` is a function we aren't using any longer.